ceph/exporter.go

```diff
@@ -39,41 +39,46 @@ type Exporter struct {
 	RbdMirror bool
 	Logger    *logrus.Logger
 	Version   *Version
+	cc        map[string]interface{}
```

Comment:
We might want to satisfy a new interface specific to ceph exporter here so that we can simplify that switch below.  Something like the following is probably good enough:

```go
type collector interface {
  Collect(chan<- prometheus.Metric, version *Version)
  Describe(/* ... */)
}
```

Reply:
done!